### ISSUE TITLE
dataset of ego_train

### ISSUE THREAD

**Issue by rtnow** [Reagan1311/Aff-Grasp#2]
Your work is excellent and very useful. However, when I run the training code, the training data should have a depth folder and files in the dataset directory, but the released dataset ego_train does not include depth. I would like to train using the dataset you released. How should I adjust or handle this ego_train dataset?

class TrainData(data.Dataset):
    def __init__(self, data_root, resize_size=476, crop_size=448, depth_gray=True):
        self.resize_size = resize_size
        self.crop_size = crop_size
        self.data_root = os.path.join(data_root, 'ego_train')
        self.data_list = []
        for i in os.listdir(self.data_root):
            if i.endswith('-img.jpg'):
                if depth_gray:
                    dep_file = i.replace('-img.jpg', '-img_graydepth.png')
                else:
                    dep_file = i.replace('-img.jpg', '-img_depth.png')
                label_file = i.replace('-img.jpg', '-label.png')
                
                if os.path.exists(opj(os.path.dirname(data_root), 'depth', dep_file)) and \
                    os.path.exists(opj(data_root, label_file)):                        
                        self.data_list.append((i, dep_file, label_file))

hope you can take some time to answer this question. Thank you very much.

**Comment by Reagan1311** [comment]
Thank you for your reminder! Related depth files have been uploaded to the Huggingface.
